package.json

```diff
@@ -14,6 +14,7 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
+    "format": "prettier --loglevel error --write './src/**/*.ts'",
```

Comment:
Is `prettier ` this not missing as a dev dependency? 

Also is there a way to ensure the format is valid as part of CI? I think it should be as simple as adding `npm run format` but you might need to validate it works.

Reply:
Oops I accidentally rebased it out. 

Just added a `prettier --check` to `lint`